frontend_app_office_document/src/container/CollaborativeEditionFrame.jsx

```diff
@@ -32,12 +32,19 @@ class CollaborativeEditionFrame extends React.Component {
       },
       accessToken: '',
       onlineEditorUrl: '',
-      ready: false
+      ready: false,
+      loggedUser: props.data.loggedUser
     }
   }
 
   async componentDidMount () {
-    console.log('%c<CollaboraFrame> did mount', `color: ${this.props.data.config.hexcolor}`)
+    const { props } = this
+    console.log('%c<CollaboraFrame> did mount', `color: ${this.props.data.config.hexcolor}`, props)
+    if (!this.checkUserPermissions()) {
+      this.sendGlobalFlashMessage(props.t("You don't have the permission to edit this file"))
```

Comment:
You should write "You are not allowed to edit this file"

Reply:
done